src/Calculator/Resources/en-US/Resources.resw

```diff
@@ -1469,6 +1481,22 @@
     <value>Categories DropDown</value>
     <comment>Screen reader prompt for the Categories dropdown field.</comment>
   </data>
+  <data name="NormalAlwaysOnTopButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
```

Comment:
>NormalAlwaysOnTopButton [](start = 14, length = 23)

Can you keep the names consistent with the above resource strings? Maybe EnterAlwaysOnTopButton and ExitAlwaysOnTopButton?